src/components/menu/menu.atoms.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC, HTMLAttributes } from 'react'
+import { ElMenuItemGroup, ElMenuItemGroupTitle } from './styles'
+
+export const MenuItemGroupTitle: FC<HTMLAttributes<HTMLDivElement>> = ({ children, ...rest }) => {
+  return (
+    <li>
+      <ElMenuItemGroupTitle {...rest}>{children}</ElMenuItemGroupTitle>
+    </li>
```

Comment:
no I think just so that it has the same pattern with other `li` at that time, which is just for semantic purpose and styling applied to its child.
now I think for the title better to use the `li` directly